src/android/de/martinreinhardt/cordova/plugins/hotspot/HotSpotPlugin.java

```diff
@@ -55,960 +55,967 @@ of this software and associated documentation files (the "Software"), to deal
-        if (!retVal && !action.equals("requestWriteSettings") && !action.equals("getWriteSettings")) {
-          // With Android 6.0+/API level>= 23, user can turn on/off permissions as
-          // necessary.
-          // Permissions are explicitely granted during the app runtime.
-          if (Build.VERSION.SDK_INT < 23) {
-            // can't write Settings
-            callback.error("write settings: false");
-            return false;
-          }
-        }
-        this.writeSettings = retVal;
-      } catch (Exception ignored) {
-        Log.e(LOG_TAG, "Could not perform permission check");
-        this.callback.sendPluginResult(new PluginResult(PluginResult.Status.ILLEGAL_ACCESS_EXCEPTION));
-      }
-    }
-    if (!this.hasPermissions()) {
-      PermissionHelper.requestPermissions(this, 0, HotSpotPlugin.permissions);
-      return true;
-    } else {
-      // pre Android 6 behaviour
-      return executeInternal(action, rawArgs, callback);
-    }
-    // Returning false results in a "MethodNotFound" error.
-  }
-
-  public void onRequestPermissionResult(int requestCode, String[] permissions, int[] grantResults)
-      throws JSONException {
-    for (int r : grantResults) {
-      if (r == PackageManager.PERMISSION_DENIED) {
-        this.callback.sendPluginResult(new PluginResult(PluginResult.Status.ILLEGAL_ACCESS_EXCEPTION));
-        return;
-      }
-    }
-    executeInternal(this.action, this.rawArgs, this.callback);
-  }
-
-  private void requestWriteSettings(CallbackContext callback) {
-    Intent intent = new Intent("android.settings.action.MANAGE_WRITE_SETTINGS");
-    intent.setData(Uri.parse("package:" + cordova.getActivity().getPackageName()));
-    intent.addFlags(Intent.FLAG_ACTIVITY_SINGLE_TOP);
-    // intent.
-    try {
-      cordova.getActivity().startActivity(intent);
-      callback.success();
-    } catch (Exception e) {
-      Log.e(LOG_TAG, "error starting permission intent", e);
-      callback.error("error starting permission intent");
-    }
-  }
-
-  private boolean executeInternal(String action, String rawArgs, CallbackContext callback) {
-    Log.i(LOG_TAG, "Running executeInternal() ");
-    Log.i(LOG_TAG, "     action: " + action);
-    Log.i(LOG_TAG, "     rawArgs: " + rawArgs);
-    if ("requestWriteSettings".equals(action)) {
-      threadhelper(new HotspotFunction() {
-        @Override
-        public void run(JSONArray args, CallbackContext callback) throws Exception {
-          requestWriteSettings(callback);
-        }
-      }, rawArgs, callback);
-      return true;
-    }
-
-    if ("getWriteSettings".equals(action)) {
-      final boolean temp = this.writeSettings;
-      threadhelper(new HotspotFunction() {
-        @Override
-        public void run(JSONArray args, CallbackContext callback) throws Exception {
-          if (temp)
-            callback.success(1);
-          else
-            callback.success(0);
+    public boolean hasPermissions() {
```

Comment:
Looks like the indentation for this is increased from 2 to 4, can you revert it back to 2 spaces so it matches with other projects.
- [ ] Done
- [ ] Approved